README.md

```diff
@@ -469,37 +474,189 @@ ASP.NET Integration
-  ```
+   ```csharp
+   using Sharpbrake.Http.Middleware;
+   ```
 
-  Then make sure to add `AirbrakeMiddleware` to your pipeline by updating the
-  `Configure` method:
+   Then make sure to add `AirbrakeMiddleware` to your pipeline by updating the
+   `Configure` method:
 
-  ```csharp
-  app.UseAirbrake(Configuration.GetSection("Airbrake"));
-  ```
+   ```csharp
+   app.UseAirbrake(Configuration.GetSection("Airbrake"));
+   ```
 
-  **Note:** In most cases you want to put `AirbrakeMiddleware` as a topmost
-  middleware component to load it as early as possible. However, if you use
-  `app.UseDeveloperExceptionPage` and/or `app.UseExceptionHandler`, then
-  `AirbrakeMiddleware` must be put after these components.
+   **Note:** In most cases you want to put `AirbrakeMiddleware` as a topmost
+   middleware component to load it as early as possible. However, if you use
+   `app.UseDeveloperExceptionPage` and/or `app.UseExceptionHandler`, then
+   `AirbrakeMiddleware` must be put after these components.
 
-  You can use the `IAirbrakeFeature` feature from the `Features` collection of
-  `HttpContext` to access the notifier that was initialized by the middleware:
+   You can use the `IAirbrakeFeature` feature from the `Features` collection of
+   `HttpContext` to access the notifier that was initialized by the middleware:
 
-  ```csharp
-  var airbrake = HttpContext.Features.Get<IAirbrakeFeature>().GetNotifier();
-  ```
+   ```csharp
+   var airbrake = HttpContext.Features.Get<IAirbrakeFeature>().GetNotifier();
+   ```
+
+NLog Integration
+----------------
+
+### Airbrake NLog target
+
+Airbrake NLog target is used to pass an exception from log messages to the
+Airbrake dashboard.
+
+1. Install the `Sharpbrake.NLog` package from NuGet (you can use "Package
+   Manager Console" from Visual Studio):
+
+   ```
+   PM> Install-Package Sharpbrake.NLog
+   ```
+
+2. Register the `Sharpbrake.NLog` assembly in the `<extensions />` section
+   of the [nlog.config][nlog-config] file:
+
+   ```xml
+   <extensions>
+     <add assembly="Sharpbrake.NLog"/>
+   </extensions>
+   ```
+
+3. Define the Airbrake target in the `<targets />` section and appropriate
+   routing rules in the `<rules />` section of `nlog.config`:
+
+   ```xml
+   <targets>
+     <target name="airbrake"
+             type="Airbrake"
+             projectId="113743"
+             projectKey="81bbff95d52f8856c770bb39e827f3f6"
+             environment="live"
+             ignoreEnvironments="dev"
+     />
+     <!-- other targets -->
+   </targets>
+   ```
+
+   ```xml
+   <rules>
+     <logger name="*" minlevel="Error" writeTo="airbrake" />
+     <!-- other rules -->
+   </rules>
+   ```
+
+   Note that both `projectId` and `projectKey` are required parameters. You can
+   set any configuration option supported by the Airbrake client in the declarative
+   way ([how to configure](#configuration)).
+
+   In the case if you need programmatically access `Notifier` (e.g. for setting
```

Comment:
`In the case if you need programmatically` -> ``When you need to access `Notifier` programmatically``